src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,10 +97,7 @@ private void configureBindings() {
   }
 
   public static void setAlliance() {
-    redAlliance =
-        (DriverStation.getAlliance().isEmpty())
-            ? false
-            : (DriverStation.getAlliance().get() == Alliance.Red);
+    redAlliance = DriverStation.getAlliance().orElse(Alliance.Blue) == Alliance.Red;
```

Comment:
This is a super minor change that realistically isn't going to really change anything, but I thought I'd include it as an example of how you can take advantage of the Java standard library's rich functionality to help with performance + readability.

This updated snippet does the same logic as before, but uses `Optional#orElse()` to streamline the data access.